packages/turbo-ignore/package.json

```diff
@@ -2,18 +2,39 @@
   "name": "turbo-ignore",
   "version": "0.0.39",
   "description": "",
-  "main": "index.js",
-  "bin": "index.js",
+  "homepage": "https://turborepo.org",
   "keywords": [],
   "author": "Jared Palmer",
   "license": "MPL-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/vercel/turborepo",
+    "directory": "turbo-ignore"
+  },
+  "bugs": {
+    "url": "https://github.com/vercel/turborepo/issues"
+  },
+  "bin": {
+    "turbo-ignore": "dist/index.js"
+  },
   "scripts": {
-    "build": "tsup index.js --format=cjs && cp ./package.json ./dist/package.json",
-    "release": "pnpm build && cd dist && npm publish"
```

Comment:
What happened to release script?

Reply:
This keeps the install faster by bundling deps (no second-order node_modules need to be installed)

Reply:
Ah will add it back